lib/src/util/safe_render_manager/safe_render_manager.dart

```diff
@@ -68,7 +67,7 @@ class SafeRenderManager extends Disposable {
 
   /// A list of [render] calls queued up while the component is in the process
   /// of rendering.
-  List<ReactElement> _renderQueue = [];
+  List<ReactElement?> _renderQueue = [];
```

Comment:
#nit could this be `final`?

```suggestion
  final _renderQueue = <ReactElement?>[];
```